css/properties/text-align-last.json

```diff
@@ -80,7 +80,11 @@
               }
             ],
             "ie": {
-              "version_added": false
+              "version_added": true,
```

Comment:
Let's add a `"partial_implementation": true` line here, since this limitation isn't part of the spec